.gitpod.yml

```diff
@@ -1,27 +1,51 @@
-# This configuration file was automatically generated by Gitpod.
-# Please adjust to your needs (see https://www.gitpod.io/docs/config-gitpod-file)
-# and commit this file to your remote git repository to share the goodness with others.
+# http://gitpod.io/#github.com/gnolang/gno
 
 tasks:
-  - name: Dev Shell
-    init: >
-      echo "install developer tools" &&
-      (cd misc/devdeps && make install) &&
-      echo "download dependencies" &&
-      go mod download &&
-      go install ./gnovm/cmd/gno
+  - name: Gno Shell
+    before: cd ./examples/
```

Comment:
Just curious, why cd into `examples`, only to cd out in the `init` command?

Reply:
before is always applied, for `init` AND for `command`. when using cached images on gitpod, the `init` directive is skipped.

`before` needs to be global, and then `init` is an exception.